gender_analysis/document.py

```diff
@@ -256,6 +276,10 @@ def _load_document_text(self):
             )
             raise FileNotFoundError(err) from original_err
 
+        # Remove Gutenberg header and footer. We may eventually want to make this into a parameter,
```

Comment:
If we've moved to having the test corpus pre-stripped, should we remove the commented out call?